tests/test_regrid.py

```diff
@@ -641,16 +641,12 @@ def test_preserve_bounds(self):
 
         output_ds = base.preserve_bounds(ds_with_bounds, ds_without_bounds, target)
 
-        assert "lat_bnds" in output_ds
-        assert "lon_bnds" in output_ds
```

Comment:
If my logic makes sense, then we should not expect horizontal grid bounds to be preserved.

Reply:
For some reason it says that @jasonb5 requested changes (I think because of this comment), but I think @jasonb5 made these changes in his last commit.

Reply:
Jason marked requested changes in the review before making the commit, so it will still show up as requested changes until he marks approved.

Example of how to request changes from a review:

![image](https://user-images.githubusercontent.com/25624127/190684886-f97ad4d7-18cc-42d5-8c74-61e60b4b34e2.png)

Reply:
@tomvothecoder @pochedls Yea I forgot to approve the changes after I made them. I'll get this merged in.